src/kixi/datastore/metadatastore.clj

```diff
@@ -2,22 +2,39 @@
   (:require [clojure.spec :as s]
             [kixi.datastore.schemastore :as schemastore]
             [kixi.datastore.segmentation :as seg]
-            [kixi.datastore.schemastore.conformers :as sc :refer [uuid]]))
+            [kixi.datastore.schemastore.conformers :as sc]
+            [clojure.spec.gen :as gen]))
 
 (s/def ::type #{"csv"})
-(s/def ::id uuid)
+(s/def ::id sc/uuid)
 (s/def ::parent-id ::id)
 (s/def ::pieces-count int?)
 (s/def ::name string?)
 (s/def ::size-bytes int?)
 (s/def ::source #{"upload" "segmentation"})
 (s/def ::line-count int?)
 (s/def ::header sc/bool?)
+(s/def ::user-id sc/uuid)
```

Comment:
This would be an edge on the graph... If we're going to ever programmatically create the query graph this should be same as what elise has in heimdall